api/envoy/service/ratelimit/v3/rls.proto

```diff
@@ -110,6 +112,9 @@ message RateLimitResponse {
 
     // The limit remaining in the current time unit.
     uint32 limit_remaining = 3;
+
+    // Number of seconds until reset of the current limit window.
+    google.protobuf.Duration seconds_until_reset = 4;
```

Comment:
Sorry should have caught this before but since you are using a duration now can you rename this to `duration_until_reset` and refer to duration above vs. seconds?